config/index.js

```diff
@@ -708,12 +708,6 @@ var conf = convict({
       format: String,
       env: 'SMS_API_SECRET'
     },
-    balanceThreshold: {
```

Comment:
I don't see `balanceThreshold` in either puppet-config or fxa-dev, so this should be a pretty straight forward removal.